versioned_sidebars/version-3.9-sidebars.json

```
@@ -197,51 +437,682 @@
-        "scalar-kubernetes/HowToUpgradeScalarDL",
-        "scalar-manager/overview"
+        {
+          "type": "doc",
+          "id": "backup-restore",
+          "label": "Backup and Restore Data in ScalarDL"
+        },
+        {
+          "type": "doc",
+          "id": "scalar-kubernetes/HowToScaleScalarDL",
+          "label": "Scale"
+        },
+        {
+          "type": "doc",
+          "id": "scalar-kubernetes/HowToUpgradeScalarDL",
+          "label": "Upgrade"
+        },
+        {
+          "type": "doc",
+          "id": "scalar-manager/overview",
+          "label": "Scalar Manager Overview"
+        }
       ]
     },
     {
       "type": "category",
       "label": "Reference",
       "collapsible": true,
       "items": [
-        "compatibility",
-        "scalardl-command-reference",
-        "scalardl-benchmarks/README"
+        {
+          "type": "doc",
+          "id": "compatibility",
+          "label": "ScalarDL Compatibility Matrix"
+        },
+        {
+          "type": "doc",
+          "id": "scalardl-command-reference",
+          "label": "ScalarDL Client Command Reference"
+        },
+        {
+          "type": "doc",
+          "id": "scalardl-benchmarks/README",
+          "label": "Run Benchmarks"
+        }
       ]
     },
     {
       "type": "category",
       "label": "Releases",
       "collapsible": true,
       "items": [
-        "releases/release-notes",
-        "releases/release-support-policy"
+        {
+          "type": "doc",
+          "id": "releases/release-notes",
+          "label": "Release Notes"
+        },
+        {
+          "type": "doc",
+          "id": "releases/release-support-policy",
+          "label": "Release Support Policy"
+        }
+      ]
+    }
+  ],
+
+  "docsJapanese": [
+    {
+      "type": "doc",
+      "label": "ScalarDL ドキュメンテーションホーム",
+      "id": "index"
+    },
+    {
+      "type": "category",
+      "label": "ScalarDL について",
+      "collapsible": true,
+      "items": [
+        {
+          "type": "doc",
+          "id": "overview",
+          "label": "概要"
+        },
+        {
+          "type": "doc",
+          "id": "design",
+          "label": "デザイン"
+        },
+        {
+          "type": "doc",
+          "id": "implementation",
+          "label": "実装"
+        },
+        {
+          "type": "doc",
+          "id": "requirements",
+          "label": "要件"
+        },
+        {
+          "type": "doc",
+          "id": "roadmap",
+          "label": "ロードマップ"
+        }
+      ]
+    },
+    {
+      "type": "category",
+      "label": "はじめよう",
+      "collapsible": true,
+      "items": [
+        {
+          "type": "doc",
+          "id": "getting-started",
+          "label": "ScalarDL Ledger を通じてコントラクトを実行"
+        },
+        {
+          "type": "doc",
+          "id": "getting-started-auditor",
+          "label": "ScalarDL Ledger と ScalarDL Auditor を通じてコントラクトを実行"
```

```suggestion
          "label": "ScalarDL Auditor モードを通じてコントラクトを実行"
```
Same as `sidebars.js`.

---

Addressed, as mentioned in the https://github.com/scalar-labs/docs-scalardl/pull/597/files#r1897305521 comment thread.